Test/VimCoreTest/VisualModeIntegrationTest.cs

```diff
@@ -1679,6 +1679,31 @@ public void AllSimpleSingleLine()
                     var span = new Span(_textBuffer.GetPoint(0), _textBuffer.CurrentSnapshot.Length);
                     Assert.Equal(span, _textView.GetSelectionSpan());
                 }
+
+                [WpfFact]
+                public void SelectInnerExpandAll()
+                {
+                    Create("<parent>", "<child>", "<grandchild />", "</child>", "</parent>");
+
+                    var initialPosition = _textBuffer.GetLineFromLineNumber(2).Start;
+                    _textView.MoveCaretTo(initialPosition);
+                    _vimBuffer.Process("vitat");
+                    Assert.Equal(_textBuffer.GetLineRange(1, 3).Extent, _textView.GetSelectionSpan());
+                }
+
+                [WpfFact]
+                public void EmptyTag_SelectInnerExpandInner()

```

Comment:
Can you add a `:set selection=exclusive` flavor of this test? In my testing, that results in zero-width selection after the first step.

Reply:
I've expanded the test case to cover both selection modes.  Thanks for the suggestion -- I always run "inclusive" mode and didn't realize a zero-width selection was even possible.